nautobot/ipam/models.py

```diff
@@ -661,17 +651,18 @@ def get_available_ips(self):
         child_ips = netaddr.IPSet([ip.address.ip for ip in self.get_child_ips()])
         available_ips = prefix - child_ips
 
-        # All IP addresses within a pool are considered usable
-        if self.is_pool:
+        # IPv6, pool, or IPv4 /31 sets are fully usable
+        if self.family == 6 or self.is_pool or self.prefix.prefixlen == 31:
             return available_ips
```

Comment:
Looking at https://github.com/netbox-community/netbox/pull/6545/files and https://github.com/netbox-community/netbox/commit/7f2f98885b68b9b9d7208587df3f277c22806b2b in context, I think the below might be preferable:

```suggestion
        # IPv6, pool, or IPv4 /31-32 sets are fully usable
        if self.family == 6 or self.is_pool or (self.family == 4 and self.prefix.prefixlen >= 31):
            return available_ips
```